packages/components/pages/f-loyalty/package.json

```diff
@@ -43,25 +43,31 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
+    "@justeat/f-globalisation": "1.0.0",
     "@justeat/f-services": "1.7.0",
-    "@justeat/f-globalisation": "1.0.0"
+    "jwt-decode": "3.1.2"
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.2.0",
-    "@justeat/f-trak": "0.7.1"
+    "@justeat/f-trak": "0.7.1",
+    "@braze/web-sdk": "3.3.0"
   },
   "devDependencies": {
+    "@braze/web-sdk": "3.3.0",
+    "@justeat/f-breadcrumbs": "3.1.0",
+    "@justeat/f-button": "3.2.0",
+    "@justeat/f-content-cards": "7.0.0",
```

Comment:
Should this be a peer dependency as well? Or are we no longer chunking so that's not necessary anymore?

Reply:
Just checked the answer to my own question, [looks like we're no longer chunkimh](https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue#L83-L89) so happy to go with this.